includes/derivatives.inc

```diff
@@ -74,6 +75,18 @@ function islandora_large_image_create_jp2_derivative(AbstractObject $object, $fo
         }
       }
 
+      if (variable_get('islandora_convert_rgb8', FALSE)) {
```

Comment:
What if the image is already rbg8? Do we need to run this? can we identify the color space first?